tests/schemas/test_identifier_schema.py

```diff
@@ -143,9 +147,7 @@ def test_given_and_not_allowed_scheme_valid_value():  # 2
         schema.load(valid_doi)
 
     errors = e.value.normalized_messages()
-    assert errors == {
-        'scheme': ['Invalid scheme for identifier 10.12345/foo.bar.']
-    }
+    assert errors == {'scheme': 'Invalid scheme doi.'}
```

Comment:
Hmm we have `doi` here and `DOI` above. I know there were tickets to make the UI all nice and labelled... but at